### PR TITLE
Us33-responsive hero&modal-Delnia

### PR DESCRIPTION
**Pull request by DelniaAlipour**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

#33 

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Comment by DelniaAlipour** [comment]
@mferryRV

File: client/src/components/Hero.css [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
 }
 
 .hero-text {
-	padding-left: 5.125em;
+	grid-column: 1 / span 7;
+	display: grid;
+	grid-template-rows: repeat(2, 1fr);
+	grid-template-areas:
+		"subheader"
+		"quote";
```
Comment (mferryRV) [inline client/src/components/Hero.css:22]: Nice use of grid

File: client/src/components/Hero.css [hunk 2]
```diff
@@ -38,26 +43,60 @@
...
 .hero-image img {
 	vertical-align: 0;
-	padding-right: 6em;
+	height: 100%;
+}
+@media (max-width: 768px) {
+	.hero-container {
+		display: grid;
+		grid-template-columns: 1fr;
+		grid-template-rows: auto 1fr;
```
Comment (mferryRV) [inline client/src/components/Hero.css:78]: Simpler than 10 columns, this works :+1:

File: client/src/components/ModalVideo.css [hunk 3]
```diff
@@ -32,13 +33,39 @@
...
 }
 .description-text {
 	padding-top: 30px;
 }
 .modal.fade .modal-dialog {
 	transition: none;
+}
+@media screen and (max-width: 768px) {
+	.modal-content {
+		width: 90%;
```
Comment (mferryRV) [inline client/src/components/ModalVideo.css:51]: Nice - no need to respect the grid in this scenario, I'm happy

**Review by mferryRV — APPROVED** [review]
Looks good to me (LGTM) 👍

**Pull request merged**